tests/test_smart_computer_player.py

```diff
@@ -0,0 +1,43 @@
+from ttt.players.smart_computer_player import SmartComputerPlayer
+from ttt.game.game import Game
+from ttt.game.board import Board
+
+
+def test_will_always_pick_the_center_space_if_available():
+    board = Board(['O','-','-','-','-','-','-','-','-'])
+    player_1 = SmartComputerPlayer('Player 1', 'X')
+    player_2 = SmartComputerPlayer('Player 2', 'O')
+    game = Game(player_1, player_2, board)
```

Comment:
Hmm, my issue with having a `MockGame` is that sometimes you use `MockGame` and sometimes `Game`, but you pass it board with certain spacer available. I'm not sure what's the key to understanding where one should use `Game`, not `MockGame` and not the other way around.

Reply:
Agreed - I think my mocks have got to a position of being a bit confused - I'm planning to take a look tomorrow at streamlining them.